src/main/java/com/internship/frejaeidjmeterplugin/jmeter/frejaRequests/ApprovalTestMain.java

```diff
@@ -0,0 +1,49 @@
+/*
```

Comment:
Remove this comments. You can Google how to setup Netbeans to stop generate this by default